server/controllers/board.js

```diff
@@ -95,3 +95,13 @@ exports.updateBoard = asyncHandler(async (req, res, next) => {
     return res.json(error);
   }
 });
+
+exports.reorderBoard = asyncHandler(async (req, res) => {
+  //to be completed
+
+  const board = await Board.findById(req.params.id);
+
+  if (!board) res.status(404).json({ error: "Board not found" });
```

Comment:
Just for consistency across the project, we are utilizing async handler to deal with the rendering of errors. So a `throw new Error('Board not found')` would be more inline with the rest of the project

Reply:
I'll **consistify** the error handling across all the controllers this way. Do we still need to set the status before we throw the error?

Reply:
Yes